### PR TITLE
Media manager build fixed

### PR DESCRIPTION
**Pull request by dev-gh**
Fixes media manager build for WIN/QT/Linux after WinSDL merge

### CODE REVIEW

**Comment by dev-gh** [comment]
@anosach-luxoft 
Please check and merge

File: src/components/media_manager/CMakeLists.txt [hunk 1]
```diff
@@ -156,6 +156,7 @@ if(EXTENDED_MEDIA_MODE)
         )
         list(APPEND AUDIO_SOURCES
             ${COMPONENTS_DIR}/media_manager/src/audio/from_mic_to_file_recorder_thread.cc
+            ${COMPONENTS_DIR}/media_manager/src/audio/a2dp_source_player_adapter.cc
```
Comment (anosach-luxoft) [inline src/components/media_manager/CMakeLists.txt]: @dev-gh I think this entry should be removed, shouldn't it?

File: src/components/media_manager/src/audio/from_mic_recorder_adapter.cc [hunk 2]
```diff
@@ -42,7 +42,7 @@ CREATE_LOGGERPTR_GLOBAL(logger_, "FromMicRecorderAdapter")
 
 FromMicRecorderAdapter::FromMicRecorderAdapter()
     : recorder_thread_(NULL)
-    , output_file_("default_recorded_audio.wav")
+    , output_file_("audio.8bit.wav") // default file within SDL appMain
```
Comment (anosach-luxoft) [inline src/components/media_manager/src/audio/from_mic_recorder_adapter.cc:45]: @dev-gh why hardcoded value? Maybe we should get it from profile?

File: src/components/media_manager/src/audio/from_mic_to_file_recorder_thread.cc [hunk 3]
```diff
@@ -295,17 +297,18 @@ media_manager::FromMicToFileRecorderThread::FromMicToFileRecorderThread(
     : threads::ThreadDelegate()
     , impl_(new Impl("-o", "-t", output_file, duration)) {
   LOGGER_AUTO_TRACE(logger_);
-  sleep_thread_ = FromMicToFileRecorderThreadPtr(
-      "AudioFromMicSuspend",
-      new timer::TimerTaskImpl<FromMicToFileRecorderThread>(
+  sleep_thread_ =
```
Comment (anosach-luxoft) [inline src/components/media_manager/src/audio/from_mic_to_file_recorder_thread.cc:300]: @dev-gh `FromMicToFileRecorderThreadPtr` was a smart pointer that removes timer finally, but now seems like you have a leak

File: src/components/media_manager/CMakeLists.txt (hunk 1 above)
Comment (dev-gh) [inline src/components/media_manager/CMakeLists.txt]: @anosach-luxoft 
That's fixed with ba6f3c55a9f6c5817b36f20b5a9c0f1d60ef7f44

File: src/components/media_manager/src/audio/from_mic_recorder_adapter.cc (hunk 2 above)
Comment (dev-gh) [inline src/components/media_manager/src/audio/from_mic_recorder_adapter.cc:45]: @anosach-luxoft 
It's being set from profile in media manager, this is just init value.

File: src/components/media_manager/src/audio/from_mic_to_file_recorder_thread.cc (hunk 3 above)
Comment (dev-gh) [inline src/components/media_manager/src/audio/from_mic_to_file_recorder_thread.cc:300]: @anosach-luxoft 
Fixed with 7ad3fe6

**Pull request merged**